.gitignore

```
@@ -7,3 +7,4 @@ build
 npm-debug.log
 .env
 .idea/
+.vscode
```

We should add a newline to the end of the file to follow the [POSIX standards](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206).
```suggestion
.vscode

```